server/conn.go

```diff
@@ -386,7 +387,8 @@ func parseHandshakeResponseBody(packet *handshakeResponse41, data []byte, offset
 			row := data[offset : offset+int(num)]
 			attrs, err := parseAttrs(row)
 			if err != nil {
-				log.Warn("parse attrs error:", errors.ErrorStack(err))
+				logutil.Logger(ctx).Warn("parse attrs failed", zap.Error(err),
+					zap.String("stack", errors.ErrorStack(err)))
```

Comment:
Should we use `zap.Stack("stack")` to replace `zap.String("stack", errors.ErrorStack(err))`?

Reply:
Is zap.Error(err) useless since we've already got the errorStack.

Reply:
@eurekaka `zap.Stack` is little different to `errors.ErrorStack`, `zap.Stack` get the current stack when calling log, but `errors.ErrorStack` take stack during error creation and saved in error, but maybe we could add a `logutil.ErrorStack` method? and it seems `zap.Stack`'s implemention is more efficient than our current "get stack " :D   

@XuHuaiyu   good catch

Reply:
@eurekaka @XuHuaiyu after some dig, I find that we should use[ `zap.Error`](https://github.com/uber-go/zap/blob/master/error.go#L45:6), it can print stack if its pkg/errors, but as our rule we should use `err` instead of `error` as log key, so I add a new `logutil.Error` to do it, PTAL

Reply:
Does the `message` here mean `121212`?

Reply:
yes